test/library/standard/Path/victor-ludorum/testcases_isAbsPath.chpl

```diff
@@ -13,3 +14,19 @@
    writeln(isAbsPath("\\server"));
 	 	 	 
    writeln(isAbsPath("."));
+
+   // Check to see if we correctly detect Pythons's abspath to the $CWD as an
+   // absolute path (This is to help identify platforms we don't support.)
+   writeln(isAbsPath(getPythonsAbspath()));
+
+   proc getPythonsAbspath() {
+     var command = "python -c 'import os; print(os.path.abspath(\"\"))'";
+     var sub = spawnshell(command, stdout=PIPE);
+
+     var absPath:string;
+     sub.stdout.readline(absPath);
+     absPath = absPath.strip();
+     sub.wait();
+     return absPath;
+     sub.wait();
```

Comment:
You've got a sub.wait() after the return statement, won't that never execute?

Reply:
whoopsie